docs/admin_api/rooms.md

```diff
@@ -87,7 +87,7 @@ GET /_synapse/admin/v1/rooms
 
 A response body like the following is returned:
 
-```jsonc
+```json
```

Comment:
this isn't really correct - `jsonc` was deliberate here (and elsewhere). See https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L2719.

Reply:
(that said, I don't know why switching to `json` makes `null` red - I'm pretty sure that's valid json...)

![image](https://user-images.githubusercontent.com/1389908/139672629-e27efc21-3ea5-4098-a2dd-3287402a252d.png)

Reply:
I hadn't heard of `jsonc`, assumed it was a typo. Sorry! I'll revert that now.

FWIW the `null`s aren't highlighted red for me:

![Screenshot from 2021-11-01 12-41-00](https://user-images.githubusercontent.com/8614563/139673102-6f70f697-3fa4-4e19-b198-7768bb85a240.png)

Reply:
mdbook does not support `jsonc`: https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html

Reply:
bleh, that's a shame. I guess this is better then.

Reply:
It does look like it's possible to add a custom theme: https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html#custom-theme and it may be worth adding that, but I think it's outside of the scope of this PR (esp. since it's already merged).

Reply:
agreed.